packages/sync-repo-settings/src/cli.ts

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env node
+
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import yargs = require('yargs');
+import {logger} from 'gcf-utils';
+import {Octokit} from '@octokit/rest';
+import {SyncRepoSettings} from './sync-repo-settings';
+import * as yaml from 'js-yaml';
+import {RepoConfig} from './types';
+import {readFileSync} from 'fs';
+
+interface Args {
+  file?: string;
+  branch?: string;
+  "github-token": string;
```

Comment:
If you're ok dropping `github-token` all together in favor of `process.env.GITHUB_TOKEN` I think we can do that.  If this is the pattern we want through, where it's passable as a flag, that's ok too.  Either way, looks like you need an `npm run fix` :)

Reply:
I think it's nice to have the option, but really to have it appear in the usage message.